sql_assesment.txt

```diff
@@ -0,0 +1,1786 @@
+
+1) List all the columns of the Salespeople table. 
+
+desc salespeople;
+
++-------+--------------+------+-----+---------+-------+
+| Field | Type         | Null | Key | Default | Extra |
++-------+--------------+------+-----+---------+-------+
+| SNUM  | int          | NO   | PRI | NULL    |       |
+| SNAME | varchar(100) | YES  |     | NULL    |       |
+| CITY  | varchar(100) | YES  |     | NULL    |       |
+| COMM  | int          | YES  |     | NULL    |       |
++-------+--------------+------+-----+---------+-------+
+
+
+
+2) List all customers with a rating of 100. 
+
+select CNAME from customers where rating=100;
+
```

Comment:
Please write SQL commands in uppercase. And avoid writing the full query in one line for better readability. It is advisable to follow the best practices of SQL.